hyperspace/BSONDocument.cpp

```diff
@@ -326,7 +328,8 @@ QVariant BSONDocument::value(const char *name, QVariant defaultValue) const
 
         case TYPE_STRING:
             return QVariant(QString::fromUtf8(bson_value_to_string(value, nullptr)));
-
+        case TYPE_ARRAY:
+            return listVariantValue(name);
```

Comment:
Let's try to be as much as consistent with all the other code, so empty line before and after this block.